assets/main.js

```diff
@@ -1,36 +1,26 @@
+require('./styles/app.scss')
+
 /*
  * This is entry point for application.
  *
  * Require JS will be used here to load all needed
  * dependencies (Vue at least).
  */
 document.addEventListener('DOMContentLoaded', function () {
+  function map (object, mapFn) {

```

Comment:
Is it really possible that this functionality does not exist in JS or some library that we're using? This looks a bit too much like reinventing the wheel.

Reply:
I think that holding this syntax sugar for object mapping here is fine. This function used here just to increase readability on line 51.

Reply:
This kind of function definitely exists in `lodash`. We cannot use it because it will increase build size just for one syntax-sugar function. Tree shaking of `lodash` is also complex thing that will take time to setup. 

In our packages we don't have this king of function because we are using `Object.keys(...).reduce(function () {...}, {})` in few places without syntax sugar.

Reply:
I see. This might be a point of improvement in the future. So if you ever feel like you'll need this kind of syntax shortening in other places, kindly extract. For now it's good.